qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -232,26 +238,67 @@ def _metadata(self):
     def _transpiled_circuits(self) -> List[QuantumCircuit]:
         """Override the transpiled circuits method to bring in the inst_map.
 
-        The calibrated schedules are transpiled into the circuits using the instruction
-        schedule map. Since instances of :class:`InstructionScheduleMap` are not serializable
-        they should not be in the transpile options. Only instances of :class:`Calibrations`
-        need to be serialized. Here, we pass the instruction schedule map of the calibrations
-        to the transpiler.
+        The transpilation should do the strict minimum to make the circuits hardware compatible.
+        Indeed, calibration experiments are designed with a specific gate sequence in mind. Any
+        transpiler operation that changes this gate sequence may compromise the validity of the
+        calibration experiment. Sub-classes may override this method to define their own
+        transpilation if need be.
```

Comment:
```suggestion
        .. note::
            The transpilation should do the strict minimum to make the circuits hardware compatible.
            Indeed, calibration experiments are designed with a specific gate sequence in mind. Any
            transpiler operation that changes this gate sequence may compromise the validity of the
            calibration experiment. Sub-classes may override this method to define their own
        transpilation if need be.
```

Please also override `set_transpile_options` method to raise warning and avoid updating it.

Reply:
Done here: [a704806](https://github.com/Qiskit/qiskit-experiments/pull/952/commits/a704806d22c2725ec1d24f7b0fbb1ad7bd1ac541)